generators/server/templates/server/src/security/oauth2.config.ts.ejs

```diff
@@ -4,11 +4,12 @@ const issuerUri = config.get('jhipster.security.oauth2.client.provider.oidc.issu
 const port = process.env.NODE_SERVER_PORT || config.get('server.port');
 const callback = `http://localhost:${port}/login/oauth2/code/oidc`;
 const dockerIssuerUri = process.env.NODE_CLIENT_PROVIDER_OIDC_ISSUER_URI || issuerUri;
+const openIdProtocol = 'protocol/openid-connect';
 
 let oauth2Config = {
-    authorizationURL: `${issuerUri}/protocol/openid-connect/auth`,
-    tokenURL: `${dockerIssuerUri}/protocol/openid-connect/token`,
-    logoutUrl: `${issuerUri}/protocol/openid-connect/logout`,
+    authorizationURL: `${issuerUri}/${openIdProtocol}/auth`,
+    tokenURL: `${dockerIssuerUri}/${openIdProtocol}/token`,
+    logoutUrl: `${issuerUri}/${openIdProtocol}/logout`,
     callbackURL: callback,
```

Comment:
Is it possible to just specify the issuer? That's what we do for Spring Security. It makes things easier because the authorization, token, and logout URLs are different between identity providers.

Reply:
These are only variables to build the endpoints. The user in the application.yml file specifies only the issuer: https://github.com/jhipster/jhipster-sample-app-nodejs-oauth2/blob/main/README.md#oauth-20--openid-connect

In this blueprint is supported only oauth2 for keycloak and okta, and there isn't a framework standard as Spring security. These endpoints specified in the oauth2.config.ts, are necessary to performs the login/logout using the auth.controller.ts implemented in the blueprint: 

https://github.com/jhipster/jhipster-sample-app-nodejs-oauth2/blob/main/server/src/web/rest/user.oauth2.controller.ts


and with the nodejs passport strategy: https://github.com/jhipster/jhipster-sample-app-nodejs-oauth2/blob/main/server/src/security/passport.oauth2.strategy.ts